{
    "source": "contrib/gis/gdal/error.py",
    "introduction": "",
    "requirement": [
    ],
    "descs": {

    }
}
